### ISSUE TITLE
Improve error message 

### ISSUE THREAD

**Issue by astrofrog** [astropy/astropy#9413]
### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

```python
# Put your Python code snippet here.
```

**Comment by pllim** [comment]
cc @keflavich

**Comment by keflavich** [comment]
Yep, this is another corner case we can catch.  It should be a pretty easy additional error message... I think we _do_ need an exception here describing how normalized convolution, which is the tool used to deal with NaNs, cannot be used when the kernel cannot be normalized.
